scripts/run_lm_batch.py

```
@@ -0,0 +1,157 @@
+    ]
+    body = {"model": MODEL, "messages": messages, "temperature": 0.2}
+
+    for attempt in range(4):
+        try:
+            r = s.post(f"{BASE}/chat/completions", json=body, headers=HEADERS, timeout=TIMEOUT)
+            r.raise_for_status()
+            content = r.json()["choices"][0]["message"]["content"]
+            arr = parse_array(content)
+            if arr and len(arr) == len(batch):
+                break
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)